csharp/dotnet/security/use_deprecated_cipher_algorithm.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Security.Cryptography;
+					
+public class Encryption
+{
+	public void CreateAes1() {
+		//ok: use_deprecated_cipher_algorithm
+		var key = Aes.Create();
+	}
+	
+	public void CreateAes2() {
+		//ok: use_deprecated_cipher_algorithm
+		var key = Aes.Create("ImplementationName");
+	}
+
+	public void CreateRijndael1() {
+		//ok: use_deprecated_cipher_algorithm
+		var key = Rijndael.Create();
+	}
+	
+	public void CreateRijndael2() {
+		//ok: use_deprecated_cipher_algorithm
+		var key = Rijndael.Create("ImplementationName");
+	}
+
+	public void CreateDES1() {
+		//ruleid: use_deprecated_cipher_algorithm
+		var key = DES.Create();
+	}
+	
+	public void CreateDES2() {
+		//ruleid: use_deprecated_cipher_algorithm
+		var key = DES.Create("ImplementationName");
+	}
+
+	public void CreateTripleDES1() {
+		//ok: use_deprecated_cipher_algorithm
+		var key = TripleDES.Create();
+	}
+	
+	public void CreateTripleDES2() {
+		//ok: use_deprecated_cipher_algorithm
+		var key = TripleDES.Create("ImplementationName");
```

Comment:
Triple DES is also not a recommended algorithm to my knowledge. 
E.g. the OWASP cheat sheet recommends only the use of AES for symmetric and ECC/RSA for asymmetric encryption.

Reply:
https://cheatsheetseries.owasp.org/cheatsheets/Cryptographic_Storage_Cheat_Sheet.html#algorithms

Reply:
I agree, that 3DES is not recommended to be used anymore but it is considered to be still kind of acceptable.
So flagging would be fine, but that would require a separate rule with a lower severity rating (3DES should only be `INFO` imo).

Reply:
Ok, that makes sense to me!